cmake/features.cmake

```diff
@@ -124,6 +124,7 @@ endif()
 
 ie_dependent_option(NGRAPH_ONNX_FRONTEND_ENABLE "Enable ONNX FrontEnd" ON "protoc_available" OFF)
 ie_dependent_option(NGRAPH_PDPD_FRONTEND_ENABLE "Enable PaddlePaddle FrontEnd" ON "protoc_available" OFF)
+option(IR_FRONTEND_ENABLE "Enable IR FrontEnd" ON)
```

Comment:
Please use ie_option

Reply:
Should we change options below to ie_option?
```
option(NGRAPH_DEBUG_ENABLE "Enable output for NGRAPH_DEBUG statements" OFF)
option(ENABLE_REQUIREMENTS_INSTALL "Dynamic dependencies install" ON)
```

Reply:
done.